web/src/lib/components/shared-components/side-bar/recent-albums.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { getAssetThumbnailUrl } from '$lib/utils';
+  import { getAllAlbums, type AlbumResponseDto } from '@immich/sdk';
+  import { handleError } from '$lib/utils/handle-error';
+  import { t } from 'svelte-i18n';
+
+  let albums: AlbumResponseDto[] = $state([]);
+
+  onMount(async () => {
+    try {
+      const allAlbums = await getAllAlbums({});
+      albums = allAlbums.slice(0, 3);
+    } catch (error) {
+      handleError(error, $t('failed_to_load_assets'));
+    }
+  });
+</script>
+
+{#if albums.length > 0}
```

Comment:
You don't need this `if`. If albums is empty the for each loop will simply have no iteration, resulting in the same